solidity/random-beacon/test/RandomBeacon.GroupCreation.test.ts

```diff
@@ -1558,23 +1659,60 @@ describe("RandomBeacon - Group Creation", () => {
             })
           })
         })
+
+        context("with correct dkg result submitted", async () => {
+          beforeEach(async () => {
+            // Set the sortition pool to determine an actual group members list.
+            const actualGroupMembers = signers
+            await (sortitionPool as SortitionPoolStub).setSelectGroupResult(
+              genesisSeed,
+              actualGroupMembers.map((signer) => signer.id)
+            )
+
+            // The result submitter submits a the actual group members list
+            // which makes the submitted DKG result valid.
+            await signAndSubmitDkgResult(
+              randomBeacon,
+              groupPublicKey,
+              actualGroupMembers,
+              startBlock,
+              noMisbehaved
+            )
+          })
+
+          it("should revert with 'unjustified challenge' error", async () => {
+            await expect(randomBeacon.challengeDkgResult()).to.be.revertedWith(
+              "unjustified challenge"
```

Comment:
We need to lower or upper-case all revert messages one day. Not something for this PR, just complaining as always.